File: OOP V1/1_class_object.cpp
#include<bits/stdc++.h>

using namespace std;

class A{
    public:
    int value;
    //constructor
    A(int value){
        this->value = value;
        cout<<"Constructor Called"<<endl;
    }

    //method
    void display(){
        cout<<"Hello\n";
    }

    //destructor
    ~A(){
        cout<<"Desonstructor Called"<<endl;
    }
};
                    
int main(){
    A obj(8);
    obj.display();
    // obj.value = 5;
    cout<<obj.value;
            
              
    return 0;
}

File: OOP V1/1_class_object.py
class A:
    value = 0
    def __init__(self,value) -> None:
        self.value = value
        print("I am Constructor")

    def display(self):
        print("Hello")

    def __str__(self) -> str:
        return "I am Class A"
    def __del__(self):
        print("I am Destructor")

obj = A(5)
obj.display()
# obj.value = 5
# print(obj.value)

# print(obj)
